src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -259,7 +259,7 @@ private static string GenerateAssemblyId(Type type)
             return sb.ToString();
         }
 
-        internal static bool GenerateSerializerFile(XmlMapping[] xmlMappings, Type[] types, string defaultNamespace, Assembly assembly, Hashtable assemblies, string codePath)
+        internal static bool GenerateSerializerStream(XmlMapping[] xmlMappings, Type[] types, string defaultNamespace, Assembly assembly, Hashtable assemblies, Stream stream)
```

Comment:
Do we need the word Stream in the method name? We're passing a stream. The old name was because it generated a file, but we're not getting a stream. Maybe change it to GenerateSerializerToStream?